src/components/form/SubmitButton.stories.tsx

```diff
@@ -0,0 +1,34 @@
+import { action } from '@storybook/addon-actions';
+import { boolean, select, text } from '@storybook/addon-knobs';
+import React from 'react';
+import { Form } from 'react-final-form';
+import { SubmitButton as SB } from './SubmitButton';
+
+export default { title: 'Components/SubmitButton' };
+
+export const SubmitButton = () => (
+  <Form onSubmit={action('onSubmit')}>
+    {({ handleSubmit }) => (
+      <form onSubmit={handleSubmit}>
+        <SB
+          spinner={boolean('Full Width', false)}
+          size={select('Size', ['small', 'medium', 'large'], 'medium')}
+          variant={select(
+            'Variant',
+            ['contained', 'text', 'outlined'],
+            'contained'
+          )}
+          fullWidth={boolean('Full Width', false)}
+          color={select(
+            'Color',
+            ['inherit', 'primary', 'secondary', 'default', 'error'],
+            'primary'
+          )}
```

Comment:
Let's try to keep the knob defaults to the same defaults as the component.
https://github.com/SeedCompany/cord-field/blob/5fb7981fde2869c50c8c551b4f953da7bcaacae0/src/components/form/SubmitButton.tsx#L50-L53